content/javascript/concepts/window-clearInterval/window-clearInterval.md

```diff
@@ -0,0 +1,40 @@
+---
+Title: 'clearInterval'
+Description: 'cancels a timer previously stablished by setInterval()'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Parameters'
+  - 'Arguments'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**`clearInterval`** is a window method used to shut down the loop-calling process of a **setInterval()** execution when this has been called and must be stopped or aborted it.
+
+## Syntax
+
+**`clearInterval`** receives a parameter called intervalId which is returned from the previously created `setInterval()` timer.
+
+The provided sample code contains some examples of how `clearInterval` is composed:
+
+```pseudo
+clearInterval(intervalID)
+```
+
+## Example
+
```

Comment:
We need an introductory sentence describing what the example does. Also, it doesn't actually stop the timer, it just declares a function to do so.